buildbot-buildmaster/buildbot/docker-compose.yml

```diff
@@ -49,6 +51,7 @@ services:
       - "traefik.tcp.routers.buildbot.tls=true"
 
       - "traefik.tcp.services.buildbot.loadbalancer.server.port=9989"
+      - "traefik.tcp.services.buildbot.loadbalancer.terminationDelay=42"
```

Comment:
Why is this required?

Reply:
It isn't - testing purposes (TLS issue). Removed. Good catch!